former-developers.md

```diff
@@ -0,0 +1,19 @@
+The SuperDARN Radar Software Toolkit (RST) was originally written by R.J Barnes and K. Baker at the Johns Hopkins University Applied Physics Laboratory (JHUAPL). Since then, members of the international SuperDARN community have contributed to the RST by adding new features, writing documentation, improving modularity, refactoring, fixing bugs, and testing the software. Below is a list of contributors to the RST. If you believe someone is missing from this list, please contact the Data Analysis Working Group <darn-dawg@isee.nagoya-u.ac.jp>.
+
+- D. Andre, `fitacf`
+- K. Baker, original JHUAPL author
+- R.J. Barnes, original JHUAPL author
+- D.D. Billett, documentation, testing
+- E.C. Bland, DAWG Co-Chair (2019-), documentation, testing
+- W.A. Bristow, `make_grid`
```

Comment:
Bill might be able to clarify but I believe Mike wrote all of the original gridding software (in IDL); maybe Bill or Rob ported it to C.

Reply:
Just checked again...Bill's name appears only in one place in the source code: `gtable.c` I'll clarify with him and others listed here
```suggestion
- W.A. Bristow, GridTableMap
```

Reply:
Bill has indicated that he should be removed from this list, so I will just commit this change now...

```suggestion
```